src/shared/views/Welcome.js

```diff
@@ -68,14 +67,14 @@ class _WelcomeScreen extends React.Component<Props> {
         if (welcomeCompleted) {
             this.finishWelcomeScreens();
         } else {
-            SplashScreen.hide();
+            // SplashScreen.hide();
```

Comment:
again here: do we want the splashscreen or not?

Reply:
It's coming back very soon, but I think I'll open a separate PR for it, just so we can merge this and get back to normal life.